styleguide/source/assets/scss/00-base/_webforms.scss

```diff
@@ -1,3 +1,16 @@
+%form-scroll {
```

Comment:
I would make this a mixin. It makes it more reusable

Reply:
Sounds good.